src/source/roku_deeplinking_screen/DeeplinkingControllerVM.bs

```diff
@@ -140,6 +140,7 @@ class DeeplinkingControllerVM extends zapp.uib.ScreenVM
       m.log.info("starting deeplinking task")
       m.deeplinkingTask = m.createInstance("DeeplinkingTask", invalid, "deeplinkingTask")
       m.observeNodeField(m.deeplinkingTask, "deeplinkingData", m.onDeeplinkingTaskResult, "value", true)
+      m.observeNodeField(m.dsp2Processor.deeplinkIOC, "isStillInDeeplinkMode", m.applyDeeplinkingForFeed, "value", true)
```

Comment:
these days you can write
`m.observe(m.dsp2Processor.deeplinkIOC.isStillInDeeplinkMode, m.applyDeeplinkingForFeed, "value", true)`